test/functional/core/job_spec.lua

```diff
@@ -290,11 +288,7 @@ describe('jobs', function()
   end)
 
   it('can omit options', function()
-    if helpers.pending_win32(pending) then return end
-    neq(0, nvim('eval', 'delete(".Xtestjob")'))
-    nvim('command', "call jobstart(['touch', '.Xtestjob'])")
-    nvim('command', "sleep 100m")
-    eq(0, nvim('eval', 'delete(".Xtestjob")'))
+    ok(eval([[jobstart('echo ""')]]) > 0)
   end)
```

Comment:
Let's delete this 'can omit options' options test entirely.

Reply:
Sure. It's covered by other tests in job_spec.lua anyway.